tests/Requests/RequestTest.php

```diff
@@ -5,220 +5,115 @@
 namespace OpenEuropa\EPoetry\Tests\Requests;
 
 use GuzzleHttp\Psr7\Response;
-use OpenEuropa\EPoetry\Tests\AbstractTest;
-use OpenEuropa\EPoetry\Type\ContactPersonIn;
-use OpenEuropa\EPoetry\Type\Contacts;
+use OpenEuropa\EPoetry\Serializer\RequestsSerializer;
 use OpenEuropa\EPoetry\Type\CreateRequests;
 use OpenEuropa\EPoetry\Type\CreateRequestsResponse;
-use OpenEuropa\EPoetry\Type\LinguisticRequestIn;
-use OpenEuropa\EPoetry\Type\RequestGeneralInfoIn;
 
 /**
  * @internal
  * @coversNothing
  */
-final class RequestTest extends AbstractTest
+final class RequestTest extends AbstractRequestTest
 {
     /**
-     * Test a SOAP request.
+     * Data provider.
+     *
+     * @return mixed
      */
-    public function testRequestSending()
+    public function requestSendingCases()
     {
-        // Generate General Info.
-        $generalInfo = new RequestGeneralInfoIn();
-
-        $generalInfo->setTitle('Test')
-            ->setInternalReference('1')
-            ->setInternalTechnicalId('1')
-            ->setRequestedDeadline(new \DateTime('2020-02-02'))
-            ->setSensitive(false)
-            ->setDocumentToBeAdopted(true)
-            ->setDecideReference('decideReference')
-            ->setSentViaRUE(true)
-            ->setDestinationCode('PUBLIC')
-            ->setSlaAnnex('')
-            ->setSlaCommitment('')
-            ->setComment('')
-            ->setOnBehalfOf('')
-            ->setAccessibleTo('');
-
-        // Generate Contacts.
-        $contacts = new Contacts();
-
-        $contact = new ContactPersonIn();
-        $contact->setUserId('1');
-        $contact->setRoleCode('AUTHOR');
-        $contacts->addContact($contact);
-
-        $contact = new ContactPersonIn();
-        $contact->setUserId('2');
-        $contact->setRoleCode('EDITOR');
-        $contacts->addContact($contact);
+        return $this->getFixture('request-test.yml');
```

Comment:
Are we using the same fixture for two different tests? This might work for now but it will quickly make tests not maintainable, we should separate the two.

Reply:
Fixed.